cmd/install.go

```diff
@@ -45,43 +51,63 @@ And to see options for a specific app before installing, run:
 
 		return nil
 	}
+	appList := GetApps()
 
-	command.AddCommand(apps.MakeInstallOpenFaaS())
-	command.AddCommand(apps.MakeInstallMetricsServer())
-	command.AddCommand(apps.MakeInstallInletsOperator())
-	command.AddCommand(apps.MakeInstallCertManager())
-	command.AddCommand(apps.MakeInstallOpenFaaSIngress())
-	command.AddCommand(apps.MakeInstallNginx())
-	command.AddCommand(apps.MakeInstallChart())
-	command.AddCommand(apps.MakeInstallLinkerd())
-	command.AddCommand(apps.MakeInstallCronConnector())
-	command.AddCommand(apps.MakeInstallKafkaConnector())
-	command.AddCommand(apps.MakeInstallKubeStateMetrics())
-	command.AddCommand(apps.MakeInstallMinio())
-	command.AddCommand(apps.MakeInstallPostgresql())
-	command.AddCommand(apps.MakeInstallKubernetesDashboard())
-	command.AddCommand(apps.MakeInstallIstio())
-	command.AddCommand(apps.MakeInstallCrossplane())
-	command.AddCommand(apps.MakeInstallMongoDB())
-	command.AddCommand(apps.MakeInstallRegistry())
-	command.AddCommand(apps.MakeInstallRegistryIngress())
-	command.AddCommand(apps.MakeInstallTraefik2())
-	command.AddCommand(apps.MakeInstallGrafana())
-	command.AddCommand(apps.MakeInstallArgoCD())
-	command.AddCommand(apps.MakeInstallPortainer())
-	command.AddCommand(apps.MakeInstallTekton())
-	command.AddCommand(apps.MakeInstallJenkins())
-	command.AddCommand(apps.MakeInstallLoki())
-	command.AddCommand(apps.MakeInstallNATSConnector())
-	command.AddCommand(apps.MakeInstallOpenFaaSLoki())
-	command.AddCommand(apps.MakeInstallNfsProvisioner())
-	command.AddCommand(apps.MakeInstallRedis())
-	command.AddCommand(apps.MakeInstallOSM())
-	command.AddCommand(apps.MakeInstallKubeImagePrefetch())
-	command.AddCommand(apps.MakeInstallRegistryCredsOperator())
-	command.AddCommand(apps.MakeInstallGitea())
+	for _, app := range appList {
+		command.AddCommand(app.Installer())
+	}
 
 	command.AddCommand(MakeInfo())
 
 	return command
 }
+
+func GetApps() map[string]ArkadeApp {
```

Comment:
This nested format is going to be hard to diff.

Please change it to add each item to the map once initialised.

```
	apps := map[string]ArkadeApp{}
        apps["mongodb"] = NewArkadeApp("mongodb", apps.MakeInstallMongoDB, apps.MongoDBInfoMsg) 
        return apps
```